src/Item/Item.service.ts

```diff
@@ -375,24 +377,27 @@ export class ItemService {
         : collection
 
       const catalystItems = await peerAPI.fetchWearables<Wearable>([urn])
-      if (catalystItems.length > 0) {
-        item = Bridge.mergeTPItem(dbItem, collection, catalystItems[0])
-      }
+      item = Bridge.mergeTPItem(
```

Comment:
do you think this validation should be kept? 

`catalystItems.length > 0`

Reply:
If there's no catalyst item, the `mergeTPItem` function will receive it as undefined, producing a full item without it.